src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheChampion/boss_argent_challenge.cpp

```diff
@@ -428,6 +437,7 @@ class npc_memory : public CreatureScript
     {
         npc_memoryAI(Creature* creature) : ScriptedAI(creature)
         {
+            instance = creature->GetInstanceScript();
```

Comment:
if you plan to use GetInstanceScript() then you have to update https://github.com/xerkoss/TrinityCore/blob/trial_of_the_champion/src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheChampion/boss_argent_challenge.cpp#L508 too